server/api/support.py

```diff
@@ -308,6 +309,7 @@ def get_jurisdiction(jurisdiction_id: str):
             auditName=jurisdiction.election.audit_name,
             auditType=jurisdiction.election.audit_type,
             online=jurisdiction.election.online,
+            deletedAt=isoformat(jurisdiction.election.deleted_at),
```

Comment:
I noticed a discrepancy with the client-side types and the data returned by the server. The client-side types indicate that the jurisdiction retrieval endpoint response includes `deletedAt` (if populated) on the jurisdiction `election` object:

https://github.com/votingworks/arlo/blob/70024cc33f5715da262774eb92e7bd771ad4e715/client/src/components/SupportTools/support-api.tsx#L41-L42

https://github.com/votingworks/arlo/blob/70024cc33f5715da262774eb92e7bd771ad4e715/client/src/components/SupportTools/support-api.tsx#L24

But the endpoint wasn't actually providing the field